p256/src/arithmetic/projective.rs

```diff
@@ -665,4 +669,9 @@ mod tests {
             assert_eq!(res.y.to_bytes(), coords.1.into());
         }
     }
+
+    #[test]
+    fn projective_to_bytes() {
+        assert_eq!([0; 33], ProjectivePoint::identity().to_bytes().as_slice());
+    }
```

Comment:
It would be good to make a note that this test is specifically for the identity point, and that the result is technically/deliberately an invalid SEC1 encoding.

Reply:
I hope this is enough:
https://github.com/RustCrypto/elliptic-curves/blob/0ba5440254df0928c69b80cf6e74182931182141/p256/src/arithmetic/projective.rs#L674-L675